numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1886,13 +1941,22 @@ io_pack(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
 {
     PyObject *obj;
     int axis = NPY_MAXDIMS;
-    static char *kwlist[] = {"a", "axis", NULL};
+    static char *kwlist[] = {"in", "axis", "order", NULL};
+    const char * c = NULL;
```

Comment:
Nit: `*` attaches to the token to the right in most of our code, would be good to be consistent.

Reply:
changed, it is now `char c`